data-table/table.js

```diff
@@ -157,6 +157,42 @@ function addColumns(numberOfColumns, index) {
      */
 }
 
+/**
+ * This function takes the id of the list container for the column names
+ * It then adds to the list another space for another column at the bottom of the list
+ * @param config id of the table
+ */
+function addColumn(config) {
+    let container = document.getElementById(config.tableIds.entryBoxDivId);
+    let child = document.createElement("DIV");
+
+    child.innerHTML = "Enter "+ config.datumConfig.names + ":";
+    container.appendChild(child);
+}
+
+/**
+ * This function enables the user to enter the name of the column,
+ * When the container with the id is clicked, an input field is created that enables the user
+ * to input the name of the column.
+ * If the user focuses out of the input field, the name is updated appropriately, and the
+ * input element removed
+ * @param config the id of the table
+ */
+
+function enterColumnValue(config) {
```

Comment:
I'd love for us to have consistent function names.

Currently for any function that creates an HTML element I've been calling it: `function create<whatever>` so this might be `function createColumnInput`

Totally happy to collaborate on what exactly those conventions are, if you'd like, let me know!